docs/using-guidelines.asciidoc

```diff
@@ -14,20 +28,20 @@ practices.
 * Map as many fields as possible to ECS.
 
 [float]
-==== Guidelines for writing fields
+===== Guidelines for writing fields
 
 * All fields must be lower case
 * Combine words using underscore
 * No special characters except `_`
 
 [float]
-==== Guidelines for naming fields
+===== Guidelines for naming fields
 
 * *Present tense.* Use present tense unless field describes historical information.
 * *Singular or plural.* Use singular and plural names properly to reflect the field content. For example, use `requests_per_sec` rather than `request_per_sec`.
 * *General to specific.* Organise the prefixes from general to specific to allow grouping fields into objects with a prefix like `host.*`.
 * *Avoid repetition.* Avoid stuttering of words. If part of the field name is already in the prefix, do not repeat it. Example: `host.host_ip` should be `host.ip`.
```

Comment:
Should we delete this rule?  Seems like between our defined field sets and other naming guidelines, this will never come into effect, and in fact, we almost contradict it with `host.hostname`

Reply:
I've reworded the clause. The gist is: we want to avoid repetition, but it's ok to make exceptions if it would be confusing not to use the conventional name.